bindings/python/py_src/safetensors/torch.py

```diff
@@ -80,7 +88,26 @@ def _view2torch(safeview) -> Dict[str, torch.Tensor]:
         result[k] = arr
 
 
-def _tobytes(tensor: torch.Tensor) -> bytes:
+def _tobytes(tensor: torch.Tensor, name: str) -> bytes:
+
+    try:
+        if not tensor.is_contiguous():
+            raise ValueError(
+                f"""You are trying to save a non contiguous tensor: `{name}` which is not allowed. It either means
+            you are trying to save tensors which are reference of each other in which case it's recommended to save
+            only the full tensors, and reslice at load time, or simply call `.contiguous()` on your tensor to pack it
+            before saving."""
```

Comment:
```suggestion
                f"You are trying to save a non contiguous tensor: `{name}` which is not allowed. It either means "
                "you are trying to save tensors which are reference of each other in which case it's recommended to save "
                "only the full tensors, and reslice at load time, or simply call `.contiguous()` on your tensor to pack it "
                "before saving."
```
If using triple quotes here, the new lines and indentation will show with the warning. It's better to use multiple lines like the suggestion.